main/src/ca/uwaterloo/flix/language/phase/typer/PrimitiveEffects.Methods.json

```diff
@@ -1,6 +1,13 @@
 {
   "methods": {
     "java.lang.Boolean::getBoolean": "Env, IO",
+    "java.lang.Character::toString": "IO",
+    "java.lang.Character::codePointAt": "IO",
+    "java.lang.Character::codePointBefore": "IO",
+    "java.lang.Character::toChars": "IO",
+    "java.lang.Character::codePointCount": "IO",
+    "java.lang.Character::offsetByCodePoints": "IO",
+    "java.lang.Character::isDefined": "FsRead, IO",
```

Comment:
Not sure if this is right but the doc mentions a unicode file: https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/lang/Character.html#isDefined(char)

Reply:
Maybe this is already loaded in the JVM so it is not really `FsRead`

Reply:
Same thing for methods below

Reply:
I wonder if "The Unicode Datafile" is actually a "file file". I think it is more likely it is something builtin-- but I am not 100% sure.

Reply:
If there is actually such a file we should probably discuss it in the paper.